build/serve.js

```diff
@@ -102,7 +102,7 @@ gulp.task('serve:watch', ['watch'], serveDevelopmentMode);
 /**
```

Comment:
spawn-backend uses the binary from .tmp. 

In case of 'serve:prod' the binary from dist should be used, I assume. 

In addition serve:prod starts browser-sync. I think, serve:prod should be 're-implemented' to use the dashboard binary from dist, only.